Rakefile

```diff
@@ -12,6 +12,12 @@ task :formulae do
   sh "brew", "ruby", "script/generate.rb"
 end
 
+desc "Dump Linux formulae data"
+task :formulae_linux do
+  ENV["HOMEBREW_FORCE_HOMEBREW_ON_LINUX"] = "1"
```

Comment:
insert `ENV["HOMEBREW_NO_COLOR"] = "1"` (see Homebrew/formulae.brew.sh#168)

Reply:
or could consider setting it globally for the whole file?